src/graph/KokkosGraph_graph_color.hpp

```diff
@@ -211,7 +211,14 @@ void d2_graph_color(
     }
 
     default:
+    {
+      color_view_type colors_out = color_view_type("Graph Colors", num_rows);
```

Comment:
@crtrott Just to check, Serial here refers strictly to the algorithms style of execution, the exec_space of the View does not need to be Kokkos::Serial?

Reply:
Yeah exactly.